sqlalchemy_redshift/dialect.py

```diff
@@ -745,62 +748,71 @@ def _get_all_relation_info(self, connection, **kw):
         return relations
 
     @reflection.cache
-    def _get_all_column_info(self, connection, **kw):
+    def _get_schema_column_info(
```

Comment:
```suggestion
    # We fetch column info an entire schema at a time to improve performance when
    # reflecting schema for multiple tables at once.
    def _get_schema_column_info(
```

We should probably leave some artifact of the discussion of performance here.